src/octod/api/releases.d

```diff
@@ -41,6 +45,7 @@ public void createRelease ( ref HTTPConnection connection, ref Repository repo,
     json["name"] = title;
     json["body"] = content;
     json["target_committish"] = tag;
+    json["prerelease"] = prerelease == Yes.prerelease ? true : false;
```

Comment:
`json["prerelease"] = (prerelease == Yes.prerelease);` ?

Reply:
.. and your question is what?

Reply:
Oh.. yeah, I see. I suppose my version is more verbose and thus easier to understand at a first glance?

Reply:
Maybe, though for me it is causing unnecessary "wait, what is going on here?" moment :)